go.mod

```diff
@@ -4,41 +4,45 @@ go 1.18
 
 require (
 	github.com/livekit/protocol v1.5.10
-	github.com/pion/ice/v2 v2.3.11
-	github.com/pion/interceptor v0.1.18
+	github.com/pion/ice/v4 v4.0.3
+	github.com/pion/interceptor v0.1.37
 	github.com/pion/logging v0.2.2
 	github.com/pion/rtcp v1.2.14
-	github.com/pion/rtp v1.8.1
-	github.com/pion/stun v0.6.1
-	github.com/pion/transport/v2 v2.2.3
-	github.com/pion/webrtc/v3 v3.2.11
+	github.com/pion/rtp v1.8.9
+	github.com/pion/stun/v3 v3.0.0
+	github.com/pion/transport/v3 v3.0.7
+	github.com/pion/webrtc/v4 v4.0.3
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.8.4
+	github.com/stretchr/testify v1.10.0
 	golang.org/x/exp v0.0.0-20230626212559-97b1e661b5df
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
```

Comment:
nit: can this be merged with the require block above?

Reply:
it is format by `go mod tidy`, do we need to merge it manually?

Reply:
yeah, you merge the blocks manually then re-run `go mod tidy` and it will sort them for you